### DISCUSSION TITLE
Download is disallowed in Chrome

### DISCUSSION THREAD

**Discussion by KuznetsovVN** [codesandbox/codesandbox-client#5935] [General] [answered]
## Description of the problem
I'm trying to download a file from codesandbox page but it didn't.
There is a warning message in the console window: "Download is disallowed. The frame initiating or instantiating the download is sandboxed, but the flag ‘allow-downloads’ is not set. See https://www.chromestatus.com/feature/5706745674465280 for more details."

![FaleSaver](https://user-images.githubusercontent.com/16776723/123937871-4aa09f00-d99f-11eb-9dd3-c0972719c365.png)

## To Reproduce

Open this example
https://codesandbox.io/s/react-filesaver-example-wc1k9

Windows, Chrome - Version 91.0.4472.114 (Official Build) (64-bit)

**Comment by sbinlondon** [comment, marked as answer]
Closing since this should be discussed/worked on in the [issue](https://github.com/codesandbox/codesandbox-client/issues/5936).
